atmo/static/js/jobs.js

```diff
@@ -1,13 +1,31 @@
 $(function() {
-    var root = this;
-    var $holder = $("#notebook-holder");
-    var content = $("#notebook-content").val();
+  $.fn.atmoNotebook = function(options) {
+    var settings = $.extend({
```

Comment:
Ah, true, removing it, this was a leftover from a previous iteration of that jQuery plugin